postgres_mixin/alerts/postgres.libsonnet

```diff
@@ -120,6 +120,207 @@
               severity: 'warning',
             },
           },
+          {
+            alert: 'PostgresHasTooManyRollbacks',
+            annotations: {
+              description: 'PostgreSQL has too many rollbacks on {{ $labels.cluster }} for database {{ $labels.datname }} with a value of {{ $value }}',
+              summary: 'PostgreSQL has too many rollbacks.',
+            },
+            expr: |||
+              avg without(pod, instance)
+              (rate(pg_stat_database_xact_rollback{db_name!~"template.*|^$"}[5m]) /
+              (rate(pg_stat_database_xact_commit{db_name!~"template.*|^$"}[5m])+ rate(pg_stat_database_xact_rollback{db_name!~"template.*|^$"}[5m]))) > 0.10
```

Comment:
Another nit here, the description has a generic "too many rollbacks", while the query is hardcoded with `> 0.10`

We should probably communicate to users exactly what percentage is being exceeded, and/or allow them to configure this like our other newer integrations via the config file